components/brave_wallet/common/hex_utils_unittest.cc

```diff
@@ -122,36 +122,40 @@ TEST(HexUtilsUnitTest, HexValueToUint256) {
 TEST(HexUtilsUnitTest, HexValueToInt256) {
   int256_t out;
   ASSERT_TRUE(HexValueToInt256("0x", &out));
-  ASSERT_EQ(out, (int256_t)0);
+  EXPECT_EQ(out, int256_t(0));
   ASSERT_TRUE(HexValueToInt256("0x0", &out));
-  ASSERT_EQ(out, (int256_t)0);
+  EXPECT_EQ(out, int256_t(0));
   ASSERT_TRUE(HexValueToInt256("0x1", &out));
-  ASSERT_EQ(out, (int256_t)1);
+  EXPECT_EQ(out, int256_t(1));
   ASSERT_TRUE(HexValueToInt256("0x1234", &out));
-  ASSERT_EQ(out, (int256_t)4660);
+  EXPECT_EQ(out, (int256_t)4660);
   ASSERT_TRUE(HexValueToInt256("0xB", &out));
-  ASSERT_EQ(out, (int256_t)11);
+  EXPECT_EQ(out, (int256_t)11);
 
```

Comment:
seems like we should have a test case here for 2^256 for https://github.com/brave/brave-core/pull/12384/files#r817177122 ?

Reply:
we should also have a test case for the max valid value

Reply:
we already have tested max, min and overflow cases below this block